src/content/documentation/enterprise/enterprise-distribution.md

```diff
@@ -207,15 +185,15 @@ It is impossible to disable loading add-ons from the profile directory.
 Use the following logic to set the values of `extensions.autoDisableScopes` and `extensions.enabledScopes` programmatically to ensure add-ons are installed automatically:
 
  1. Edit the [administrative config](https://support.mozilla.org/products/firefox-enterprise/policies-customization-enterprise/policies-overview-enterprise) file.
- 2. Check for the presence of lines that set the `extensions.autoDisableScopes` and/or `extensions.enabledScopes` preferences and replace/add them as needed.
- 3. These preference lines should be used like below, with values of your choice as explained in the top of this section:
+ 2. Check for lines that set the `extensions.autoDisableScopes` and `extensions.enabledScopes` preferences and replace or add them as needed.
+ 3. These preference lines should be used as below, with values of your choice as explained at the start of this section:
 
 <!-- Syntax Highlighting -->
 
 ```js
 defaultPref("extensions.autoDisableScopes", 0);
 defaultPref("extensions.enabledScopes", 15);
-// Or use binary value like this
+// Or use a binary value like this
 defaultPref("extensions.enabledScopes", 0b1111);
 ```
 
```

Comment:
@mkaply any idea weather the note in the following lines is still relevant? If it is, I think we should reword it as a statement of the functionality, otherwise, remove.